packages/core/import-map.importmap

```diff
@@ -66,6 +66,8 @@
     "@cds/core/internal-components/close-button/register.js": "/base/dist/core/internal-components/close-button/register.js",
     "@cds/core/internal-components/overlay": "/base/dist/core/internal-components/overlay/index.js",
     "@cds/core/internal-components/overlay/register.js": "/base/dist/core/internal-components/overlay/register.js",
+    "@cds/core/internal-components/panel": "/base/dist/core/internal-components/panel/index.js",
+    "@cds/core/internal-components/panel/register.js": "/base/dist/core/internal-components/panel/register.js",
```

Comment:
@mathisscott  What I was actually missing to run the unit tests. I thought at first I needed only to specify the `index.js` then I saw the overlay also specified the `register.js` and turns out this was the missing piece.